src/release/issue-triaging.md

```diff
@@ -1,125 +1,276 @@
+      earlier issue, or explicitly link to the duplicate issue.
+    - If you are not sure, you can still leave a comment to indicate the other
+      issue is possibly a duplicate, similar, or related.
 
-For applying and removing labels, unprivileged users can use **@rustbot** to add or remove
-[the labels allowed by the `triagebot.toml` configuration](https://github.com/rust-lang/rust/blob/master/triagebot.toml).
-For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos -needs-triage`.
+For applying and removing labels, unprivileged users can use **@rustbot** to add
+or remove [the labels allowed by the `triagebot.toml`
+configuration](https://github.com/rust-lang/rust/blob/master/triagebot.toml).
 
-To see a list of all labels, check out the "labels" page next to the search bar in the issue tracker.
+For example, `@rustbot label +T-compiler +C-bug +A-linkage +O-macos
+-needs-triage`.
 
-[URLO]: https://users.rust-lang.org
-[IRLO]: https://internals.rust-lang.org/
-[Discord]: https://discord.gg/rust-lang
+To see a list of all labels, check out the "labels" page next to the search bar
+in the issue tracker.
 
-### ICE Triage
-For [Issues that have both I-ICE and needs-triage](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AI-ICE%20label%3Aneeds-triage)
-* Check that the issue is actually an ICE, and not more accuratly described with `I-crash` or `I-hang`.
-* If it is an older (like latest stable) version of rust, ask for or check the latest nightly.
-* Check for duplicates, but don't close as duplicate unless you're sure they represent the same underlying issue.
-  Prefer simply linking to the issue as possibly related/duplicate.
-* If it does not have a reproduction, comment asking for one and add S-needs-repro. if there isn't one for around a month it should generally be closed.
-* If the reproduction is not minimal, add `E-needs-mcve` or create a Minimal Complete and Verifible Example yourself.
-* Add `A-*` labels based on the code that causes the issue (check backtraces!),
-  and the nature of the repro (eg. if the repro is a weird trait impl or the backtrace points to `rustc_trait_selection`, add `A-traits`)
-* Add `T-*`, `WG-*`, `PG-*`, `F-*`, `requires-*`, and `regression-*` labels as appropriate.
+### Labels
 
+There are many different labels that can be applied to issues.
 
-## Further triaging
+- `needs-triage`: Signals that an issue is new and needs initial triage.
+- [`T-*`]: Specifies the team or teams that this issue is relevant to. For
+      example `T-compiler`, `T-types` or `T-libs`.
+- [`WG-*`]: Specifies the working groups that this issue is relevant to, for
+      example `WG-debugging`.
+- [`PG-*`]: Specifies the project groups that this issue is relevant to, for
+      example the `PG-exploit-mitigations`.
+- [`C-*`]: Specifies the category of the label, for example a bug, tracking
+      issue or discussion.
+    - `A-diagnostics`: issues usually don't have any `C-*` label.
```

Comment:
It's perhaps surprising for an `A-*` item to be the first in a list of `C-*` items.  Also, in general, this line reads rather ambiguously.

Doing a quick [search](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AA-diagnostics&page=1), quite a lot of issues labels `A-diagnostics` do also have `C-*` labels.

Reply:
It's a bit weird because we don't do this consistently, at all. Issue templates for diagnostics issues don't add `C-bug` by default, but you can manually add it. Also some issues involve diagnostics but are actually bugs e.g. ICEs. I'll add some stuff to clarify this.

Reply:
Moved under `A-*` and described this behavior due to new diagnostics issue template.